### PR TITLE
Include gsutil creds

### PR DESCRIPTION
**Pull request by tejohnso**
@fjvallarino please review

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -37,8 +37,11 @@ deployment:
...
     commands:
+      - git clone git@github.com:Rise-Vision/private-keys.git
+      - mv private-keys ..
+      - gcloud auth activate-service-account 452091732215@developer.gserviceaccount.com --key-file ../private-keys/storage-server/rva-media-library-ce0d2bd78b54.json
       - ./node_modules/.bin/cross-env NODE_ENV=prod npm run build
       - mvn appengine:update -Dappengine.version=r$(echo -n $((CIRCLE_BUILD_NUM%15))) -Dappengine.appId=rvashow2
       - mvn appengine:set_default_version -Dappengine.version=r$(echo -n $((CIRCLE_BUILD_NUM%15))) -Dappengine.appId=rvashow2
-      - create-local-viewer
-      - upload-local-viewer
+      - ./create-local-viewer
```
Comment (fjvallarino) [inline circle.yml]: @tejohnso should we use "sh ./create-local-viewer.sh" instead, since the file is probably not marked as executable?

Comment (tejohnso) [inline circle.yml]: @fjvallarino It's marked executable in the repo so I think it should work https://github.com/Rise-Vision/viewer/blob/master/create-local-viewer.sh

Comment (fjvallarino) [inline circle.yml]: @tejohnso what about the missing extension? I just tried on sh, bash and zsh and running the script without ".sh" failed for me.

Comment (tejohnso) [inline circle.yml]: @fjvallarino updated with extension

**Comment by fjvallarino** [comment]
:+1:

**Pull request merged**